src/Shared/BuildEnvironmentHelper.cs

```diff
@@ -563,8 +570,8 @@ public BuildEnvironment(BuildEnvironmentMode mode, string currentMSBuildExePath,
                 var existsCheck = mode == BuildEnvironmentMode.VisualStudio ? new Func<string, bool>(_ => true) : File.Exists;
```

Comment:
I seem to recall `f => File.Exists(f)` is more performant than `File.Exists`. Don't care that much.